src/main/java/androidx/graphics/Bitmap.kt

```diff
@@ -111,3 +116,56 @@ inline fun createBitmap(
 ): Bitmap {
     return Bitmap.createBitmap(width, height, config, hasAlpha, colorSpace)
 }
+
+
+/**
+ * Returns ByteArray compressed from this bitmap with the specified [format]
+ * and [quality].
+ *
+ * @param format The format of bitmap.
+ * @param quality Hint to the compressor, 0-100. 0 meaning compress for small size, 100 meaning compress for max quality.
+ * @return ByteArray
+ */
+inline fun Bitmap.toByteArray(format:CompressFormat = CompressFormat.JPEG, @IntRange(from = 0, to = 100) quality: Int=100)
```

Comment:
I still don't find compressing to a byte array useful. Compressing directly into a stream seems the more common case